src/Compilers/VisualBasic/Portable/Symbols/MethodSymbol.vb

```diff
@@ -747,6 +747,18 @@ Namespace Microsoft.CodeAnalysis.VisualBasic.Symbols
         ''' </remarks>
         Friend MustOverride Function CalculateLocalSyntaxOffset(localPosition As Integer, localTree As SyntaxTree) As Integer
 
+        ''' <summary>
+        ''' Specifies whether original variables (corresponding to proxies) are preserved during lambda rewriting.
```

Comment:
Consider adding more details to this comment and probably examples. People who are not familiar with the context might have a hard time to understand the intent.